docs/introduction/diffusion_intro/diffusion_intro.rst

```diff
@@ -0,0 +1,138 @@
+At the moment, this method is the default in the kymotracker.
+
+2) **Gaussian refinement (recommended)**
+
+Gaussidan refinement fits a Gaussian function to improve the position estimate. This method is better at refining tracks that are close together. 
+Using Gaussian refinement to improve the position estimate of tracks is always recommended. Gaussian refinement can also be used to refine missing frames, but this is not the default. When doing MSD analysis, 
+you have to activate the refinement of missing frames, by setting `refine_missing_frames = True` (see section on :ref:`gaussian refinement<kymotracker_gaussian_refinement>`).
+
+
+FAQ on diffusion analysis
+--------------------------
+- **The diffusion coefficients for some of the tracks are negative, what does that mean?**
+    Sometimes a diffusion coefficient of a single track is negative. For example, when your diffusion coefficient is low, you will have some diffusion coefficients above and some below zero. 
+    The negative values do not have a biological meaning by themselves, but the average of all the diffusion coefficients from different tracks should still give a good, positively values estimate of the diffusion 
+    coefficient (provided that you have enough tracks, and that the protein is diffusing freely). The negative data points *should not* be removed from the dataset, otherwise you get a bias in your estimate. 
+
+- **What happens to the diffusion coefficient if a protein bumps into another protein or against other obstacles?**
+    If there are many diffusive proteins binding to the DNA at once and they are not able to pass each other, the proteins will confine the motion, resulting in sub-diffusive or confined diffusive behavior.
+    If the DNA is held between two beads in an optical tweezer experiment, the beads can also confine the motion of a diffusive protein, if the protein diffuses far enough. 
+    If you are interested in free diffusion of a protein, it is best to keep the density of protein binding events low enough, so the proteins don't meet often. 
+    This will help reducing the number binding events, but not when the motion is so wide that the protein ends up on the beads. If you want to exclude the effect of the beads, you can consider cutting the track 
+    to exclude the part where the protein reaches the bead.
+
+- **How do I account for the motion of the DNA molecule itself?**
+    Though the DNA in an optical tweezer experiment is constrained, it still is fluctuating. The motion of the DNA sets a lower limit on what diffusion coefficient can be observed for a protein diffusing 
+    along the DNA. To find out this lower limit, you can use a static molecule as a reference. The motion of the DNA also can lead to a bias in the estimation of the diffusion coefficient :cite:`vestergaard2014optimal`.`
+    (SHOULD WE MENTION THE BIAS HERE, HOW BIG IS THE BIAS?)
```

Comment:
I would say the MSD of a static molecule gives a flat graph (there's a small slope in the first few tau points). the flat line crosses the y-axis at certain level and this could be the estimate of the bias.

Reply:
hmm, yes, I guess a static molecule would undergo a kind of subdiffusive motion. At small distances, the DNA can diffusive freely, but at larger distances, it will feel the confinement from the beads. How this affects the MSD of a moving protein depends on how floppy the DNA is and how that compares to the time resolution and the diffusion coefficient of the protein of interest. Vestergaard PRE 2014 discusses various limits, but does not explain the bias in an intuitive way, so I don't know a good way to summarize this for now. Most papers take the approach you did, to use the fluctuation of a static protein as a lower limit. I think I'll just say that this is a common thing to do.